org.openmw.OpenMW.json

```diff
@@ -0,0 +1,360 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
```

Comment:
not a blocker, but does this work on the more recent 3.26 runtime or are there dependencies that lock it here?